docs/bigcommerce-for-wordpress/setup/multi-site.md

```diff
@@ -30,7 +30,7 @@ Multiple sites can share the same API credentials or you can choose to create a
 
 ## Getting your API credentials
 
-1. To get your store's API credentials, log into your BigCommerce store and head to `Advanced Settings` > `API Accounts`. Then, click the blue `Create API Account` button on the top left hand side.
+To get your store’s API credentials, log into your BigCommerce store and head to `Advanced Settings` > `API Accounts`. Then, click the blue `Create API Account` button on the top left-hand side.

```

Comment:
```suggestion
1. To get your store's API credentials, log into your BigCommerce store and head to `Advanced Settings` > `API Accounts`.

```
There's a list of steps here, and you've removed 1, but left 2-4.  There's also duplication b/t 1 & 2.  This suggested change works in tandem with a suggested edit to 2 below.